GraphView/po/pl-local.po

```diff
@@ -0,0 +1,634 @@
+
+#: GraphView/graphview.py:684
+msgid "Show avatars"
+msgstr "Pokaż awatary"
+
+#: GraphView/graphview.py:687 GraphView/graphview.py:3575
+msgid "Highlight the home person"
+msgstr "Wyróżnij osobę główną"
+
+#: GraphView/graphview.py:691 GraphView/graphview.py:3583
+msgid "Show full dates"
+msgstr "Pokaż pełne daty"
+
+#: GraphView/graphview.py:695 GraphView/graphview.py:3591
+msgid "Show places"
+msgstr "Pokaż miejsca"
+
+#: GraphView/graphview.py:698 GraphView/graphview.py:2877
+msgid "Default"
+msgstr "Domyślny"
+
+#: GraphView/graphview.py:704
+msgid "Place format"
+msgstr "Format miejsc"
+
+#: GraphView/graphview.py:709 GraphView/graphview.py:3599
+msgid "Show tags"
+msgstr "Pokaż tagi"
+
+#: GraphView/graphview.py:711
+msgid "Layout"
+msgstr "Layout"
+
+#: GraphView/graphview.py:729
+msgid "Person theme"
+msgstr "Motyw osoby"
+
+#: GraphView/graphview.py:734
+msgid "Path color to home person"
+msgstr "Kolor ścieżki do osoby domowej"
+
+#: GraphView/graphview.py:738
+msgid "Font:"
+msgstr "Czcionka:"
+
+#: GraphView/graphview.py:752
+msgid "Avatars style"
+msgstr "Styl awatarów"
+
+#: GraphView/graphview.py:758
+msgid "PNG files"
+msgstr "Plik PNG"
+
+#: GraphView/graphview.py:763
+msgid "Male avatar:"
+msgstr "Awatar męski:"
+
+#: GraphView/graphview.py:764
+msgid "Choose male avatar"
+msgstr "Wybierz męski awatar"
+
+#: GraphView/graphview.py:776
+msgid "Female avatar:"
+msgstr "Awatar damski:"
+
+#: GraphView/graphview.py:777
+msgid "Choose female avatar"
+msgstr "Wybierz damski awatar"
+
+#: GraphView/graphview.py:789
+msgid "Themes"
+msgstr "Motywy"
+
+#: GraphView/graphview.py:802 GraphView/graphview.py:3609
+msgid "Show animation"
+msgstr "Pokaż animację"
+
+#: GraphView/graphview.py:806
+msgid "Animation speed (1..5 and 5 is the slower)"
+msgstr "Szybkość animacji (1..5 gdzie 5 jest wolniejsze)"
+
+#: GraphView/graphview.py:810
+msgid "Animation count (0..8 use 0 to turn off)"
+msgstr "Liczba animacji (0..8 gdzie 0 wyłącza)"
+
+#: GraphView/graphview.py:819
+msgid "Animation"
+msgstr "Animacja"
+
+#: GraphView/graphview.py:833
+msgid "Search in all database"
+msgstr "Szukaj we wszystkich bazach"
+
+#: GraphView/graphview.py:835
+msgid "Also apply search by all database."
+msgstr "Zastosuj również wyszukiwanie według wszystkich baz."
+
+#: GraphView/graphview.py:838
+msgid "Show person images"
+msgstr "Pokaż zdjęcia osób"
+
+#: GraphView/graphview.py:841
+msgid "Show persons thumbnails in search result list."
+msgstr "Pokaż miniatury osób na liście wyników wyszukiwania."
+
+#: GraphView/graphview.py:844
+msgid "Show bookmarked first"
+msgstr "Pokaż najpierw dodane do zakładek"
+
+#: GraphView/graphview.py:847
+msgid "Show bookmarked persons first in search result list."
+msgstr "Pokaż zaznaczone osoby jako pierwsze na liście wyników wyszukiwania."
+
+#: GraphView/graphview.py:849
+msgid "Search"
+msgstr "Szukaj"
+
+#: GraphView/graphview.py:880
+msgid "Select a dot file name"
+msgstr "Wybierz nazwę pliku kropki"
```

Comment:
GraphView requires GraphViz and the native file format for GraphViz is 'dot'. Let's keep the original file format name: "Wybierz nazwę pliku dot"

Reply:
You're right. Correct.